README.md

```diff
@@ -1,3 +1,155 @@
-![AlphaESS Monitor Logo](/alphaess_monitor/icon.png)
+# Home Assistant Addon: Alpha ESS to MQTT
+## A simple way to get the current monitoring data from [alphaess.com](https://alphaess.com) via MQTT
 
-Monitor for the AlphaESS Solar Module
\ No newline at end of file
+## About
+
+This Addon is based on the reverse engineered API alphaess.com uses to show your current metrics on their web interface / mobile app. The obtained data that is sent via MQTT to the configured broker (or directly to the Home Assistant instance this addon is running on).
+
+## Installation
+
+1. Add `https://github.com/DasLetzteEinhorn/AlphaESS_Monitor_Hass` as a new repository.
+2. Install the Addon
+3. Add your Username and Password (used on the alphaess.com webinterface) in the corresponding fields and save your configuration.
+4. Start the Addon
+
+## Configuration
+
+Default configuration:
+
+```yaml
+log_level: info
+base_topic: alphaess
+timeout: 60
+username: <your username>
+password: <your password>
+```
+
+By default, the addon connects to your Home Assistant's broker (Mosquitto), on which the addon is installed. If the broker is installed somewhere else, the following settings must be added:
```

Comment:
Maybe you can add a little more information to Mosquitto or add an resource where you can read more about it.